### DISCUSSION TITLE
CVE-2022-21713? is this an issue?

### DISCUSSION THREAD

**Discussion by dbones** [prometheus-operator/kube-prometheus#1632] [Q&A] [answered]
https://github.com/grafana/grafana/security/advisories/GHSA-63g3-9jq3-mccv

from what I can see this repo make use of version 8.3.4 which is impacted, and the advisory needs it to be 8.3.5

[from what i can see here](https://github.com/prometheus-operator/kube-prometheus/security) - its not being picked up

am i correct and this needs to be raised as an issue?

**Comment by metalmatze** [comment, marked as answer]
Yep, your assumptions are right. 
It won't be found as part of code scanning it seems.

We have a bot that sends weekly (I think) updates to all our versions, so there should be an automated PR soon that will bump the version. As it's just the default version I think we can wait until then. For people that want to run the patch version right away, there's always the option to set it explicitly to 8.3.5.

**Reply by metalmatze** [reply]
I manually started our update bot that @paulfantom wrote :tada: 
Otherwise we would have waited until Monday. 
It created https://github.com/prometheus-operator/kube-prometheus/pull/1633 which patches the default Grafana to 8.3.6.

**Reply by ArthurSens** [reply]
merged :)
